openlibrary/templates/merge_queue/merge_queue.html

```diff
@@ -129,62 +120,87 @@ <h1>$_('Community Edit Requests')</h1>
+                  $:_('MR #%(id)s opened %(date)s by @%(submitter)s', id=r['id'], date=datestr(r["created"]), submitter=r["submitter"])
+                  $if is_open:
+                    $if is_submitter:
+                      <a class="mr-close-link" title="Click to close this Merge Request" data-mrid="$r['id']" data-merge-type="$r['mr_type']" href="javascript:;">&times;</a>
+                </span>
+              </div>
             </td>
-            <td id="reviewer-cell-$(r['id'])">
-              $if can_merge and is_open and r.get('reviewer'):
-                $r['reviewer'] <span class="mr-unassign" data-mrid="$r['id']" data-merge-type="$r['mr_type']">&times;</span>
-                <br>
-                $datestr(r['updated'])
-              $elif r.get('reviewer'):
-                $r['reviewer']
-            </td>
-            <td>
-              $if is_open:
-                $if is_submitter:
-                  <a class="mr-close-link" data-mrid="$r['id']" data-merge-type="$r['mr_type']" href="javascript:;">$_('Close')</a>
-                $elif can_merge:
+            <td class="review-cell">
+              <div id="reviewer-cell-$(r['id'])" class="review-cell__unassign">
+                $if can_merge and is_open and r.get('reviewer'):
+                  $r['reviewer'] <span class="mr-unassign" data-mrid="$r['id']" data-merge-type="$r['mr_type']">&times;</span>
+                $if not is_open:
+                  $if r.get('reviewer'):
+                    <img class="review-cell__check" src="/static/images/icons/icon_check.png"><span class="reviewer">@$r['reviewer']</span>
+              </div>
+              <div class="reviewer-btn">
+                $if is_open and can_merge:
                   $ show_link = not r.get('reviewer') or r.get('reviewer') == username
                   $ extra_classes = '' if show_link else ' hidden'
-                  <a class="mr-resolve-link$extra_classes" id="mr-resolve-link-$r['id']" data-mrid="$r['id']" data-merge-type="$r['mr_type']" href="$url" target="_blank">$_('Merge')</a>
+                  <button id="mr-resolve-btn-$r['id']"class="review-cell__review">
+                    <strong>
+                      <a class="mr-resolve-link" id="mr-resolve-link-$r['id']" data-mrid="$r['id']" data-merge-type="$r['mr_type']" data-url="$url" href="$url" target="_blank"><span class="WHITE">$_('REVIEW')</span></a>
+                    </strong>
+                  </button>
+                </div>
+              </td>
+              <td class="mr-comment-review-cell">
+                <button id="mr-comment-btn" class="mr-comment-review-cell__comments mr-comment-btn-$r['id']">
+                  <a href="javascript:;" class="comment-expand"
+                  data-hidden-comments="hidden-comments-$r['id']"
+                  data-latest-comment="latest-comment-$r['id']"
+                  data-btn-comments="mr-comment-btn-$r['id']"
+                  data-old-comments="old-comments-section-$r['id']">
```

Comment:
TODO: Find a better way to reference elements in client-side code.

Reply:
I'll do some research to try to find a better solution for this.

Reply:
I've got it from here.  We talked about this.